docs/reference/migrating_versions.rst

```diff
@@ -44,7 +44,7 @@ Some specific changes:
 
   - `expectations_store_name`
   - `evaluation_parameter_store_name`
-  - `profiling_store_name`
+  - `validations_store_name`
```

Comment:
I'm not sure this is right. @abegong : you really really wanted validations gone as a required option in favor of always happening in actions, right? If that's still the case, we need to keep profiling store name for now, but deprecate it as soon as profiling can be configured separately from the datacontext (since we've talked about the option that data_context does not have profiling as a core concern)